Content.Shared/_RMC14/Xenonids/Parasite/SharedXenoParasiteSystem.cs

```diff
@@ -411,6 +411,18 @@ public override void Update(float frameTime)
             if (_net.IsClient)
                 continue;
 
+            // 20 seconds before burst, spawn the larva
+            if (infected.BurstAt - TimeSpan.FromSeconds(20) < time && infected.SpawnedLarva == null)
```

Comment:
The 20 seconds should be moved to a data field at some point but it's fine for now
```suggestion
            if (infected.BurstAt - TimeSpan.FromSeconds(20) < time && infected.SpawnedLarva == null)
```

Reply:
made this a datafield